test/testmetrics/api-beta/00-install.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  labels:
+    app.kubernetes.io/name: keptnmetricsprovider
+    app.kubernetes.io/instance: keptnmetricsprovider
+    app.kubernetes.io/part-of: keptn-lifecycle-toolkit
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: metrics-operator
+  name: keptnmetricsprovider
+spec:
+  type: "prometheus"
+  targetServer: "http://prometheus-k8s.monitoring.svc.cluster.local:9090"
+  secretKeyRef:
+    name: "prometheusLoginCredentials"
+    key: "my-credentials"
+# v1beta1,KeptnMetric cannot be applied to the cluster due to the presence of
+# conversion webhook for KeptnMetric CRD
+# The non-existence of conversion path from v1beta1 to v1alpha3 (HUB)
+# causes the system to refuse this CR when applied
```

Comment:
Please read the comment. We have 2 options here, either create the conversion path from `v1beta1` to `v1alpha3` (and remove it in the next PR) or accept the fact that `v1beta1.KeptnMetric` is currently in the state, where it cannot be applied to the cluster